build.gradle

```diff
@@ -9,11 +12,11 @@ apply plugin: "blackbaud-templates"
 apply plugin: "integration-test"
 
 dependencies {
-    compile gradleApi()
+    compileOnly gradleApi()
     compile localGroovy()
     compile 'com.google.guava:guava:18.0'
     compile 'org.eclipse.jgit:org.eclipse.jgit:4.0.2.201509141540-r'
-    compile 'org.gradle:gradle-tooling-api:2.1'
+    compile "org.gradle:gradle-tooling-api:${parseGradleVersionFromWrapperProps()}"
```

Comment:
if something goes wrong and this method throws an exception, the error might be eaten by the dependency stuff... i'd extract this to an ext variable